src/app/components/account/account-avatar/account-avatar.tsx

```diff
@@ -4,25 +4,9 @@ import { Box, CircleProps } from 'leather-styles/jsx';
 
```

Comment:
You can prob take this out of it's own folder since it is a single file? The UI refactor may have removed files here and we missed this?